content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -221,7 +221,7 @@ A structural schema is an [OpenAPI v3.0 validation schema](#validation) which:
 
 Non-structural example 1:
 
-```yaml
+```none
```

Comment:
The commit message doesn't explain this change - at least, I couldn't work out the motivation.

Reply:
These snippets are not YAML, they are just snippets of YAML plus some dots. They cannot be used in place of a valid YAML file. We don't need to syntax highlight them as YAML. Just rendering them as plain code would suffice.

Reply:
I recommend explaining that in the commit message and PR description (doesn't have to be a long explanation).

Reply:
Done.